src/components/EditorWidgets/Date/DateControl.js

```diff
@@ -23,13 +24,16 @@ export default class DateControl extends React.Component {
 
     /**
      * Set the current date as default value if no default value is provided. An
-     * empty string means the value is intentionally blank.
+     * empty string means the value is intentionally blank. 
+     * Default time can be now or 0:00 am.
      */
     if (!value && value !== '') {
-      this.handleChange(new Date());
+      const { defaultTime = moment().startOf('day').toDate() } = this.props;
```

Comment:
`moment().startOf('day').toDate()` sets to midnight adjusted to the current timezone (i.e. in `-07` timezone, MST, the time saved will be `T07:00:00.000Z`). @drlogout Is that how you think it should be, or should we set it to `00:00` UTC?